code/mob/living/life/critical.dm

```diff
@@ -2,6 +2,8 @@
 /datum/lifeprocess/critical //for mobs that use crit (humans only right now)
 	process(var/datum/gas_mixture/environment)
 		var/mult = get_multiplier()
+		if(owner && HAS_MOB_PROPERTY(owner, PROP_TEMP_CRIT))
+			mult *= get_temperature_mult()
```

Comment:
Hm. I don't really like this - crit multipler is something that I feel should be purely lag-compensation. Seek a second opinion on this perhaps